app/experimenter/nimbus-ui/src/components/PreviewURL/index.tsx

```diff
@@ -19,24 +20,39 @@ const CopyableURL: React.FC<CopyableURLProps> = ({
   branch,
   collection,
 }) => {
+  // This about: url cannot be linked from web content, so help with auto-copy.
   let url = `about:studies?optin_slug=${experimentSlug}&optin_branch=${branch}`;
   const clipboard = navigator.clipboard;
   if (collection) {
     url = `${url}&optin_collection=${collection}`;
   }
-  return <code onClick={() => clipboard.writeText(url)}>{url}</code>;
+  return (
+    <code onClick={() => clipboard.writeText(url)} style={{ cursor: "copy" }}>
```

Comment:
We've generally been avoiding inline styles for reuse so you could pull this out and add `.cursor-copy { cursor: copy }` to our `index.scss` and then use `className` instead. I think we still need an issue for this but we talked about copying JSON blobs to the clipboard so we could potentially use this class when that feature is worked on!